source/analysis/AM3DAdditionAB.cpp

```diff
@@ -253,5 +257,6 @@ void AM3DAdditionAB::computeCachedValues() const
             cachedData_[j] += data.at(j);
     }
 
+    cachedDataRange_ = AMUtility::rangeFinder(cachedData_);
```

Comment:
I wasn't sure if invalid values were something that were implemented by default; if there's no invalid value, then there's no need to have that argument. Gotcha.